server/fishtest/static/html/maintenance.html

```diff
@@ -58,7 +58,7 @@
   <body>
     <article>
       <div class="header">
-        <img src="/img/stockfish.png" alt="logo" />
+        <img src="/img/stockfish.webp" alt="logo" />
```

Comment:
Weird, the maintenance page loads the pgn but does not find the webp.

Reply:
mh.. works for me?
![grafik](https://github.com/user-attachments/assets/b17a3001-125b-4e93-a692-353577be3cf0)

Reply:
but yeah not on dev.. https://dfts-0.pigazzini.it/, though not even the any of the (static) assets https://dfts-0.pigazzini.it/img/stockfish.webp

Reply:
More weirdness, not it works, and the mime type is correct, before it was html/txt ...
![image](https://github.com/user-attachments/assets/a1449f74-52f6-41cf-b595-b274e5bef5c2)

Reply:
In a sandbox the webp is not loaded and the mime type is wrong.

![image](https://github.com/user-attachments/assets/eefcc531-fccb-4ab6-af3f-709ab8d7cf0e)

Reply:
Do the other static files load ? 
Does it work with master ?

Reply:
I fixed a bug in the maintenance nginx conf. The pgn was loaded for serendipity because already cached.

Reply:
IMO you can delete the png, we don't have a fallback in place.